packages/next-server/lib/router/router.ts

```diff
@@ -206,7 +206,7 @@ export default class Router implements IRouterInterface {
       // Add the ending slash to the paths. So, we can serve the
       // "<page>/index.html" directly for the SSR page.
       // @ts-ignore this is temporarily global (attached to window)
-      if (__NEXT_DATA__.nextExport) {
+      if (!process.env.__NEXT_EXPORT_TRAILING_SLASH && __NEXT_DATA__.nextExport) {
```

Comment:
Same as https://github.com/zeit/next.js/pull/6664/files#r265789907

Shouldn't this condition be without a `!`. If it's true it has to add the trailing slash right.

Reply:
Right, updated